# HG changeset patch
# User Rik <rik@octave.org>
# Date 1345744372 25200
#      Thu Aug 23 10:52:52 2012 -0700
# Node ID e3dc9ff8e0f2619ca93db5ba79d074d0b968e350
# Parent  a83b7b2f95eef833de412f3155241d8a64e1e127
polyeig.m: Use @code, not @var, macro to silence warnings from texinfo.

* polyeig.m: Use @code, not @var, macro to silence warnings from texinfo.

diff --git a/scripts/polynomial/polyeig.m b/scripts/polynomial/polyeig.m
--- a/scripts/polynomial/polyeig.m
+++ b/scripts/polynomial/polyeig.m
@@ -13,26 +13,27 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{z} =} polyeig (@var{C0}, @var{C1}, @dots{}, @var{Cl})
-## @deftypefnx {Function File} {[ @var{v}, @var{z} ] =} polyeig (@var{C0}, @var{C1}, @dots{}, @var{Cl})
+## @deftypefnx {Function File} {[@var{v}, @var{z}] =} polyeig (@var{C0}, @var{C1}, @dots{}, @var{Cl})
 ##
 ## Solve the polynomial eigenvalue problem of degree @var{l}.
 ##
-## Given a @var{n*n} matrix polynomial @var{C(s)} = @var{C0 + C1 s +
-## @dots{} + Cl s^l} polyeig solves the eigenvalue problem (@var{C0} +
-## @var{C1} + @dots{} + @var{Cl})v = 0. Note that the eigenvalues
-## @var{z} are the zeros of the matrix polynomial. @var{z} is a
-## @var{lxn} vector and @var{v} is a @var{(n x n)l} matrix with columns
-## that correspond to the eigenvectors.
+## Given an @var{n*n} matrix polynomial
+## @code{@var{C}(s) = @var{C0} + @var{C1} s + @dots{} + @var{Cl} s^l}
+## polyeig solves the eigenvalue problem
+## @code{(@var{C0} + @var{C1} + @dots{} + @var{Cl})v = 0}.
+## Note that the eigenvalues @var{z} are the zeros of the matrix polynomial.
+## @var{z} is an @var{lxn} vector and @var{v} is an (@var{n} x @var{n})l matrix
+## with columns that correspond to the eigenvectors.
 ##
 ## @seealso{eig, eigs, compan}
 ## @end deftypefn
 
 ## Author: Fotios Kasolis
 
 function [ z, varargout ] = polyeig (varargin)
   
